### PR TITLE
RATEST-122: Fix Edit Provider Test

### PR DESCRIPTION
**Pull request by kdaud**
Ticket ID: https://issues.openmrs.org/browse/RATEST-122
Description: 
Resurrecting Edit Provider Test in the legacy ui

### CODE REVIEW

File: ui-tests/src/test/java/org/openmrs/reference/EditProviderTest.java [hunk 1]
```diff
@@ -27,14 +27,14 @@
     private String providerUuid;
 
     @Before
-    public void setUp() {
+	public void setUp() throws Exception {
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/EditProviderTest.java:30]: any reason for adding the throws here?

File: ui-tests/src/test/java/org/openmrs/reference/page/ProviderPage.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
     private static final By IDENTIFIER_FIELD = By.cssSelector("#content input[name=\"identifier\"]");
     private static final By PERSON_FIELD = By.id("person_id_selection");
     private static final By SAVE = By.cssSelector("#content input[type=\"submit\"]");
```
Comment (kaweesi) [inline ui-tests/src/test/java/org/openmrs/reference/page/ProviderPage.java:19]: you can change all the rest to make this page readable else no need of this single one

**Comment by kdaud** [comment]
There should be some issues with github. After triggering a commit, it closed the PR and does not give a provision of Re-openning. This is the second time to observe this scenario But this happens when the PR is not updated with the upstream master. cc: @kaweesi

**Comment by kdaud** [comment]
Refer to this [commit](https://github.com/openmrs/openmrs-distro-referenceapplication/pull/404)

**Pull request closed without merging**